tap_google_ads/streams.py

```diff
@@ -7,6 +7,8 @@
 from singer import utils
 from google.protobuf.json_format import MessageToJson
 from google.ads.googleads.errors import GoogleAdsException
+from google.api_core.exceptions import ServerError, TooManyRequests
```

Comment:
Looking at the stack trace in [TDL-19194](https://jira.talendforge.org/browse/TDL-19194), it looks like we have:

```
2022-05-24 16:14:32,922Z    tap - CRITICAL 500 Internal error encountered. [type_url: "type.googleapis.com/google.ads.googleads.v10.errors.GoogleAdsFailure"
2022-05-24 16:14:32,922Z    tap - CRITICAL value: "\022\026EDI3x6aWReFYRgzvDSvAVA"
2022-05-24 16:14:32,922Z    tap - CRITICAL ]
2022-05-24 16:14:32,922Z    tap - Traceback (most recent call last):
... more of the stack trace

2022-05-24 16:14:32,924Z    tap - The above exception was the direct cause of the following exception:

... the stack trace for the above referenced exception
2022-05-24 16:14:32,925Z    tap - google.api_core.exceptions.InternalServerError: 500 Internal error encountered. [type_url: "type.googleapis.com/google.ads.googleads.v10.errors.GoogleAdsFailure"
2022-05-24 16:14:32,925Z    tap - value: "\022\026EDI3x6aWReFYRgzvDSvAVA"
```

Do you know if these `GoogleAdsFailure` errors are caught by these exceptions?

Reply:
From the log, it seems like in the tap side, we would get `google.api_core.exceptions.InternalServerError` and `GoogleAdsFailure` are caught by `InternalServerError`.

The following line is just the part of a message in the `InternalServerError` exception,

`[type_url: "type.googleapis.com/google.ads.googleads.v10.errors.GoogleAdsFailure"  value: "\022\026EDI3x6aWReFYRgzvDSvAVA"]`. 

We have verified the same kind of error in our local environment,

![image](https://user-images.githubusercontent.com/88327452/172839431-c1c8bac9-5016-4fea-b128-a7cdf2d5b917.png)

Reply:
Awesome!